docs/rector_rules_overview.md

```diff
@@ -14158,6 +14158,26 @@ Classes that have no children nor are used, should have abstract
 
 <br><br>
 
+### `MoveVariableDeclarationNearReferenceRector`
+
+- class: [`Rector\SOLID\Rector\Variable\MoveVariableDeclarationNearReferenceRector`](/rules/solid/src/Rector/Variable_/MoveVariableDeclarationNearReferenceRector.php)
+- [test fixtures](/rules/solid/tests/Rector/Variable_/MoveVariableDeclarationNearReferenceRector/Fixture)
+
+Variable declarations should be moved directly above the first usage of those variables
+
+```diff
+ function foo()
+ {
+     -$var = 1;
+     if (mktime() === false) {
+        +$var = 1;
+         return $var;
+     }
+ }
+```
+
+<br><br>
```

Comment:
This looks like manual... is this generated with `composer docs` command?

Reply:
I copied it from the section above.

Reply:
No need to. The `composer docs` generates this feil